packages/scaffolding-cli/templates/src/csr/README.md

```diff
@@ -0,0 +1,49 @@
+# Client Side Rendering Web Application
+
+## Implementation details
+
+The template is a Client Side Rendered (CSR) implementation using:
+
+- [React.js](https://reactjs.org/)
+- [Jest](https://jestjs.io/)
+- [Cypress](https://docs.cypress.io/)
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br /> Open
+[http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br /> You will also see any lint errors
+in the console.
+
+### `npm run test`
+
+Launches the test runner in the interactive watch mode.<br /> See the section
```

Comment:
html and referencing their documentation.

Reply:
what did you mean?

Reply:
did you mean Jest docs?

Reply:
<br />

Reply:
The docs should be ours, this is copy past e with html in the md.